setup.py

```diff
@@ -87,6 +87,9 @@ def run_tests(self):
     zip_safe=False,
     packages=find_packages(exclude=['examples', 'tests']),
     include_package_data=True,
+    install_requires=[
+        'tzlocal',
```

Comment:
I think our policy is to not have dependencies and I'm not sure it's worth to add `tzlocal`.

Reply:
@berkerpeksag thanks, I remove `tzlocal`, and convert datetime to local time by pyhton standard library.

Reply:
I'd say wait for the result of the discussion in #1234.